scripts/testing_scripts/test_all_sandia

```diff
@@ -112,6 +112,10 @@ if [[ "$HOSTNAME" == kokkos-dev\.sandia\.gov* ]]; then
   MACHINE=kokkos-dev
 fi
 
+if [[ "$HOSTNAME" == sogpu01.sandia.gov* ]]; then
```

Comment:
I thought the point of the wildcard `*` that you don't need to spell out `sandia.gov`
Also you escaped the dot above for other machines.  Please confirm that you don't need to.

Reply:
@dalg24 I updated the PR to remove .sandia.gov, thanks
cf060480bbc92f46900db2d563327fa3f7d94d9b